includes/functions.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Miscellaneous functions.
+ * These are mostly utility or wrapper functions.
+ *
+ * @package Google\Web_Stories
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Web_Stories;
+
+use Google\Web_Stories\Interfaces\FieldState;
+use Google\Web_Stories\Stories_Renderer\Renderer;
+use Google\Web_Stories\Stories_Renderer\FieldState\BaseFieldState as GridView;
+use Google\Web_Stories\Stories_Renderer\FieldState\CarouselView;
+use Google\Web_Stories\Stories_Renderer\FieldState\CircleView;
+use Google\Web_Stories\Stories_Renderer\FieldState\ListView;
+
+/**
+ * Render stories based on the passed arguments.
+ *
+ * @param array $attrs Arguments for fetching stories.
+ * @param array $query_args Query arguments for stories.
+ *
+ * @return void
+ */
+function render_stories( $attrs = [], $query_args = [] ) {
+	$stories_obj = new Story_Query( $attrs, $query_args );
+	//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo $stories_obj->render();
+}
+
+/**
+ * Returns list of stories based on the arguments passed to it.
+ *
+ * @param array $attrs Arguments for fetching stories.
+ * @param array $query_args Query arguments for stories.
+ *
+ * @return array
+ */
+function get_stories( $attrs = [], $query_args = [] ) {
+	$stories_obj = new Story_Query( $attrs, $query_args );
+
+	return $stories_obj->get_stories();
+}
+
+/**
+ * Render stories based on customizer settings.
+ *
+ * @return void
+ */
+function render_theme_stories() {
+	//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+	echo get_plugin_instance()->customizer->render_stories();
+}
+
+/**
+ * Returns field state for the provided view type.
+ *
+ * @param string $view View Type.
+ *
+ * @return GridView|ListView|CircleView|CarouselView
+ */
+function get_field( $view = 'grid' ) {
```

Comment:
This shouldn't be a function in this file.

Let's move it to a factory class or something.

Reply:
I agree, this feels weird here.